content/tutorials/getting-started/index.md

```diff
@@ -15,21 +15,30 @@ lastmod: 2021-08-30T16:00:00+02:00
 
 In this tutorial, we'll describe every step to help you run your first test with Gatling Enterprise.
 
-{{< alert tip >}}
+{{< alert info >}}
 **Requirements**
 
 * A valid [GitHub](https://github.com/) account
 * A Java Development Kit (JDK) installed in version 8 or 11
-* The [Gatling bundle](https://gatling.io/open-source/#downloadgatling) as a starter or an existing Maven/Gradle/SBT Gatling project
+{{< /alert >}}
+
+{{< alert tip >}}
+**Recommendations**
+
+* Using the [Gatling bundle](https://gatling.io/open-source/#downloadgatling) is beginner friendly.
```

Comment:
"Using" seems weird here, I'd just keep: "The Gatling bundle is beginner friendly"

Reply:
"Using the Gatling Bundle is the beginner friendly option"
OR
"The Gatling Bundle is beginner friendly and is the best choice if you're just getting started"